lib/util/pragma.js

```diff
@@ -6,6 +6,15 @@
 
 var JSX_ANNOTATION_REGEX = /^\*\s*@jsx\s+([^\s]+)/;
 
+function getCreateClassFromContext(context) {
+  var pragma = 'createClass';
+  // .eslintrc shared settings (http://eslint.org/docs/user-guide/configuring#adding-shared-settings)
+  if (context.settings.react && context.settings.react.createClass) {
+    pragma = context.settings.react.createClass;
+  }
+  return pragma.split('(')[0];
```

Comment:
If someone sets an invalid pragma value, shouldn't the plugin error out, instead of trying to fix it for them?